Removed opcache

MPParsley opened the pull request:
Fixed typo 7.1 => 7.2;
But note that these commands result in an error:

brew install -v php72-opcache
Error: No available formula with the name "php72-opcache"
==> Searching for a previously deleted formula (in the last month)...
Warning: homebrew/core is shallow clone. To get complete history run:
  git -C "$(brew --repo homebrew/core)" fetch --unshallow

Error: No previously deleted formula found.
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
==> Searching taps on GitHub...
Error: No formulae found in taps.

MPParsley committed:
Fixed php 7.2 opcache commands

Fixed typo 7.1 => 7.2;
But note that these commands result in an error:

brew install -v php72-opcache
Error: No available formula with the name "php72-opcache"
==> Searching for a previously deleted formula (in the last month)...
Warning: homebrew/core is shallow clone. To get complete history run:
  git -C "$(brew --repo homebrew/core)" fetch --unshallow

Error: No previously deleted formula found.
==> Searching for similarly named formulae...
Error: No similarly named formulae found.
==> Searching taps...
==> Searching taps on GitHub...
Error: No formulae found in taps.

zero2one commented on documentation/Installation/PHP.md:
````diff
@@ -235,7 +235,7 @@ brew install -v php71-opcache
 ```bash
 brew unlink php71 && brew link php72
-brew install -v php71-opcache
+brew install -v php72-opcache
````
Opcode caching is (in the homebrew-core version of PHP) automatically installed. No need to install it anymore.

MPParsley committed:
Removed opcache

Opcode caching is (in the homebrew-core version of PHP) automatically installed. No need to install it anymore.
